pages/components/task_review.php

```diff
@@ -1 +1,29 @@
-<!-- page for reviewing task results -->
\ No newline at end of file
+<!-- page for reviewing task results -->
+<form method="post" action=""> <!-- review_task.php is backend -->
+  <div class="form-group">
+    <label for="task-list">Search task</label>
+    <input type="text" class="form-control" id="task-search" name="task-search">
+  </div>
```

Comment:
counter-intuitive design
Should instead list all the tasks that the dephead has assigned and improve from there